core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -817,13 +818,11 @@ var ExplorationEditorMainTab = function() {
     await action.waitForAutosave();
     await general.scrollToTop();
     var nodeStateElement = await explorationGraph.$(
-      `.e2e-test-node=${stateName}`);
+      `.e2e-test-node*=${stateName}`);
     await waitFor.visibilityOf(
       nodeStateElement,
       'State ' + stateName + ' takes too long to appear or does not exist');
-    var nodeElement = await explorationGraph.$$(
-      `.e2e-test-node=${stateName}`)[0];
-    var deleteNode = nodeElement.$(deleteNodeLocator);
+    var deleteNode = await nodeStateElement.$(deleteNodeLocator);
```

Comment:
Why are you using ` nodeStateElement` in replace of `nodeElement` above?

Reply:
Reverted.